### PR TITLE
SweetAlert confirmation for reveal

### PR DESCRIPTION
**Pull request by stian8**
Alert box for _all_ Reveals

### CODE REVIEW

File: src/components/Toolbar/index.js [hunk 1]
```diff
@@ -189,7 +190,16 @@ export default class Toolbar extends Component {
   }
 
   reveal(scopeString) {
-    this.props.onReveal(scopeString);
+    swal({
+      title: 'Are you sure you want to reveal the answer?',
+      icon: 'warning',
+      buttons: true,
+      dangerMode: true,
+    }).then((willDelete) => {
```
Comment (stevenhao) [inline src/components/Toolbar/index.js:198]: hmm, maybe this should be named something else, like `shouldReveal` (or something simple like `yes`)

**Comment by stevenhao** [comment]
Thanks for the PR! Looks great, though I had one small style question

File: src/components/Toolbar/index.js [hunk 2]
```diff
@@ -189,7 +190,16 @@ export default class Toolbar extends Component {
   }
 
   reveal(scopeString) {
-    this.props.onReveal(scopeString);
+    swal({
+      title: 'Are you sure you want to reveal the answer?',
```
Comment (stevenhao) [inline src/components/Toolbar/index.js:194]: Could we make the value of title vary based on `scopeString`. I think the current wording could be confusing to users; they might think they're revealing the entire grid.

Comment (stian8) [reply src/components/Toolbar/index.js]: Ah good point! Added and also some text :)

**Pull request merged**